src/Bridge/Symfony/Resources/config/services.xml

```diff
@@ -16,6 +16,11 @@
             <argument>%sonata.exporter.writer.xls.filename%</argument>
             <argument>%sonata.exporter.writer.xls.show_headers%</argument>
         </service>
+        <service id="sonata.exporter.writer.xlsx" class="Sonata\Exporter\Writer\XlsxWriter" public="false">
```

Comment:
IMO the service could also be loaded conditionally, otherwise I'm not sure if it will fail if the package is not installed (it's throwing an exception in the constructor)

Reply:
Do you mean by checking the if the Spreadsheet class exists in the services.xml (now services.php) and then add the service?

Reply:
Yeah, before I was thinking about doing it in the `SonataExporterExtension.php`, but now we can do it there.

Reply:
Perfect. Just fixed this 👍